Makefile.pre.in

```diff
@@ -1708,10 +1711,13 @@ PYTHON_HEADERS= \
 		$(srcdir)/Include/osdefs.h \
 		$(srcdir)/Include/osmodule.h \
 		$(srcdir)/Include/patchlevel.h \
+		$(srcdir)/Include/py_curses.h \
```

Comment:
This should be connected to the curses extension module. We have the `MODULE_{MODNAME}_DEPS` variables for this. If `git grep "^MODULE__\?[A-Z0-9]\+_DEPS"`, you'll see them all.

Reply:
Done